packages/core/src/hooks/useAfterFirstRender/__tests__/useAfterFirstRender.test.tsx

```diff
@@ -0,0 +1,35 @@
+import { renderHook } from "@testing-library/react-hooks";
+import useAfterFirstRender from "../";
+
+describe("useAfterFirstRender", () => {
+  it("should initially set isAfterFirstRender to false", () => {
+    const { result } = renderHook(() => useAfterFirstRender());
+    expect(result.current.current).toBeFalsy();
```

Comment:
Why is it `current.current`?

Reply:
since renderHook's result always return a ref, so I had to get to that ref's value, and my hook returns a ref so the ref's value is another ref